addons/hr_timesheet/views/project_sharing_views.xml

```diff
@@ -22,12 +22,12 @@
                 <page string="Timesheets" id="timesheets_tab" attrs="{'invisible': [('allow_timesheets', '=', False)]}">
                     <group>
                         <group>
-                            <div class="o_td_label">
+                            <div colspan="2">
                                 <label for="planned_hours" string="Allocated Hours" class="me-2" attrs="{'invisible': [('encode_uom_in_days', '=', True)]}"/>
                                 <label for="planned_hours" string="Allocated Days" class="me-2" attrs="{'invisible': [('encode_uom_in_days', '=', False)]}"/>
-                                <field name="planned_hours" class="oe_inline" widget="timesheet_uom_no_toggle"/>
+                                <field name="planned_hours" class="timesheet_uom_no_toggle oe_inline" widget="timesheet_uom_no_toggle"/>
```

Comment:
https://github.com/odoo/odoo/blob/2a015600b2cf4631cbf16ca0802191c0f60c93d3/addons/web/static/src/views/form/form_controller.scss#L704
can't we use that?